parts/language/dict_en.lua

```diff
@@ -292,99 +292,99 @@ FNNS and     {"Support 3",
     {"Tetris Effect",
         "tec tetriseffectconnected",
         "game",
-        "PS/Oculus Quest/Xbox/NS/Windows | Singleplayer/Multiplayer\nOr TE(C) for short. An official Tetris game with fancy graphics and soundtracks that react to your input. The basic version (without the word “Connected”) only has single-player modes. The extended version, Tetris Effect Connected, features four online battle modes, Connected (VS), Zone Battle, Score Attack, and Classic Score Attack.",
+        "PS/Oculus Quest/Xbox/NS/Windows | Singleplayer/Multiplayer\nOr TE(C) for short. An official Tetris game with elaborate graphics and soundtracks that react to the player's input. The basic version (Tetris Effect) has only single player modes. The extended version, Tetris Effect Connected, has four online battle modes: Connected (VS), Zone Battle, Score Attack, and Classic Score Attack.",
     },
     {"Tetris 99",
         "t99 tetris99",
         "game",
-        "Nintendo Switch | Singleplayer/Multiplayer\nA game famous for its 99-player battle royale mode and has many interesting strategies not present in traditional battle Tetris games. It also has limited single-player modes like Marathon and bot matches available as DLC.",
+        "Nintendo Switch | Singleplayer/Multiplayer\nA game famous for its 99-player battle royale mode and novel strategies not found in traditional battle Tetris games. The game's DLC includes limited single-player modes such as Marathon and Bot Matches.",
     },
     {"Puyo Puyo Tetris",
         "ppt puyopuyotetris",
         "game",
-        "PS/NS/Xbox/Windows | Singleplayer/multiplayer\nA game that combines two games, Tetris and Puyo Puyo, and can battle between those two games. It has many modes for single-player and online\n\n[MrZ: The Steam PC version has worse controls and horrible online experience.]",
+        "PS/NS/Xbox/Windows | Singleplayer/multiplayer\nAn official Tetris game featuring a combination of Tetris and Puyo Puyo, as well as battles between the two games. It also includes many single player and online modes.\n\n[MrZ: The Steam PC version has worse controls and poor online experience.]",
```

Comment:
> [MrZ: The Steam PC version has worse controls and **horror** online experience.]

I am thinking about reverting to the original?

Reply:
Trying to make it more neutral so